app/index.js

```diff
@@ -117,14 +118,13 @@ hrm.onreading = function() {
   // Peek the current sensor values
   const now = Date.now();
   if ((Date.now() - hrmLastUpdated) > hrmUpdateInterval) {
-      //console.log("Current heart rate: " + hrm.heartRate);
-      hrLabel.text = hrm.heartRate;
+        hrLabel.text = hrm.heartRate;
+      }
       hrmLastUpdated = now;
-  }
-};
+    };
 
-// Begin monitoring the sensor
-hrm.start();
+  // Begin monitoring the sensor
+  hrm.start();
```

Comment:
This should be done only if activity is enabled?

Reply:
I tried that approach, would involve quite a bit of refactoring as settings are not imported until after the txt file is read.